src/foam/i18n/ClientCacheTranslationService.js

```diff
@@ -14,6 +14,7 @@ foam.CLASS({
   ],
 
   imports: [
+    'theme',
```

Comment:
Should be optional import and loadTheme() should only be called if a theme exists